### PR TITLE
Bug fix: -.gch is created in cwd

### PR DESCRIPTION
**Pull request by sabel83**

### CODE REVIEW

File: test/system/include/pattern/boost_regex.hpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#if defined(__GNUC__) && !defined(__clang__)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
```
Comment (r0mai) [inline test/system/include/pattern/boost_regex.hpp]: Where was this a problem? Including boost headers with -isystem didn't silence warnings from boost?

**Comment by r0mai** [comment]
:+1: 
I don't know why the pull request flow didn't by itself.
I started it by hand, and it's green: http://ci.metashell.org/job/metashell-pullrequest-flow/22/console

File: test/system/include/pattern/boost_regex.hpp (hunk 1 above)
Comment (sabel83) [inline test/system/include/pattern/boost_regex.hpp]: They were not treated as errors, but were shown. See https://travis-ci.org/sabel83/metashell/jobs/102770239 as an example.

Comment (r0mai) [inline test/system/include/pattern/boost_regex.hpp]: Strange. Boost is included with `-isystem` which should silence all warnings from those headers. Seems like a minor compiler bug. Anyway, nice workaround :+1:

**Comment by r0mai** [comment]
Oh, and the automatic Jenkins job finally started (and finished) as well. It just needed a 15 minutes for some reason.

**Pull request merged**